backbone/src/main/java/org/researchstack/backbone/task/OrderedTaskFactory.java

```diff
@@ -0,0 +1,621 @@
+    public static final String TremorTestExtendArmStepIdentifier    = "tremor.handAtShoulderLength";
+    public static final String TremorTestBendArmStepIdentifier      = "tremor.handAtShoulderLengthWithElbowBent";
+    public static final String TremorTestTouchNoseStepIdentifier    = "tremor.handToNose";
+    public static final String TremorTestTurnWristStepIdentifier    = "tremor.handQueenWave";
+    public static final String ActiveTaskMostAffectedHandIdentifier = "mostAffected";
+    public static final String ActiveTaskLeftHandIdentifier         = "left";
+    public static final String ActiveTaskRightHandIdentifier        = "right";
+    public static final String ActiveTaskSkipHandStepIdentifier     = "skipHand";
+    // Conclusion Step Identifiers
+    public static final String ConclusionStepIdentifier     = "conclusion";
+
+    /**
+     * Returns a predefined task that measures hand tremor.
+     *
+     * In a tremor assessment task, the participant is asked to hold the device with their most affected
+     * hand in various positions while accelerometer and motion data are captured.
+     *
+     * @param context                can be app or activity, used for resources
+     * @param identifier             The task identifier to use for this task, appropriate to the study.
+     * @param intendedUseDescription A localized string describing the intended use of the data
+     *                               collected. If the value of this parameter is null, none will be used
+     * @param activeStepDuration     The duration for each active step in the task in seconds
+     * @param tremorOptionList       Options that affect which active steps are presented for this task.
+     * @param handOption             Options for determining which hand(s) to test.
+     * @param taskOptionList             Options that affect the features of the predefined task,
+     *                               conclusion option will be ignored at this time.
+     *
+     * @return An active tremor test task that can be presented with an `ORKTaskViewController` object.
+     */
+    public static NavigableOrderedTask tremorTask(
+            Context context,
+            String identifier,
+            String intendedUseDescription,
+            int activeStepDuration,
+            List<TremorTaskExcludeOption> tremorOptionList,
+            HandOptions handOption,
+            List<TaskExcludeOption> taskOptionList)
+    {
+        // Coin toss for which hand first (in case we're doing both)
+        final boolean leftFirstIfDoingBoth = (new Random()).nextBoolean();
+        return tremorTask(context, identifier, intendedUseDescription, activeStepDuration,
+                tremorOptionList, handOption, taskOptionList, leftFirstIfDoingBoth);
+    }
+
+    // This method is separate mainly for unit testing purposes, to eliminate randomness
+    protected static NavigableOrderedTask tremorTask(
```

Comment:
Is this task always the same?  Aren't some tasks defined by service responses?  Or is that only for surveys?